ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -804,9 +807,42 @@ then
 	# Try to get an USB console
 	nshterm /dev/ttyACM0 &
 else
-	mavlink start -r 800000 -d /dev/ttyACM0 -m config -x
+	# Don't start mavlink in passthrough mode because we currently can't seem to stop it.
+	if [ $PASSTHROUGH == no ]
+	then
+		mavlink start -r 800000 -d /dev/ttyACM0 -m config -x
```

Comment:
This is pretty evil as it will make it impossible to reconfigure the board. Is there an issue with a MAVLink app running on USB in the background?